packages/x-components/static-docs/build-search-ui/web-archetype-integration-guide.md

```diff
@@ -164,30 +158,58 @@ invoked at any time. Note that you should only call this function **once**.
-| isSpa                   | `boolean`                                                                            |          | True when the X Components archetype is being run on top of an SPA.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
-| &lt;extra parameters>   | `any`                                                                                |          | Any other parameter to be sent directly to the API calls. For example, some times is needed to filter the search catalog with a warehouse parameter. In that case you can just add `warehouse: <your-warehouse-identifier>` to the snippet config.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+## Notes on X Archetype integration
 
-### Callbacks
+To successfully integrate Interface&nbsp;X in your commerce store using the X&nbsp;Archetype, check out further information about:
 
-As the callbacks are a powerful tool in the initialisation options, we provide this section to get
-deep in their use. This is an example of initialisation providing callbacks to subscribe to certain
-events.
+* **Initialization options** supported in [snippet configuration](#snippet-configuration)
+* **[Callbacks and X&nbsp;event&nbsp;types](#callbacks-interface-x-events-types)** available to subscribe to when initializing
```

Comment:
Guess we're having a linking conflict here among repos. For the link to work properly in code needs the ampersand, but it won't work in eDocs. I'll replace the `&` symbol to avoid conflicts.

Reply:
Now works in my local and in eDocs